pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x,y):
+    return y - y
```

Comment:
There seems to be a typo here.
`
```suggestion
    return x - y
```